Comments/ClientApp/src/components/SubmitResponseDialog/SubmitResponseDialog.js

```diff
@@ -131,65 +131,65 @@ export class SubmitResponseDialog extends PureComponent {
 								</div>
 
 							</div>
+						</>
+						}
 
-							{respondingAsOrganisation === "yes" &&
-							<Fragment>
+						{(!isLead && respondingAsOrganisation === "yes") &&
```

Comment:
just a question... is there a good reason why respondingAsOrganisation is a string "yes" or "no", rather than just a boolean?
if there's a good reason, leave it.